### PR TITLE
Update Android Auto sensor icon based on state

### PR DESCRIPTION
**Pull request by mikosoft83**
## Summary
Adding state based icon to the sensor, servers as visual aid.

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

File: app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt [hunk 1]
```diff
@@ -90,11 +90,17 @@ class AndroidAutoSensorManager : SensorManager, Observer<Int> {
...
+            true -> "mdi:car"
+            else -> "mdi:car-off"
+        }
+
         onSensorUpdated(
             context,
             androidAutoConnected,
             connected,
-            androidAutoConnected.statelessIcon,
+            icon,
```
Comment (jpelgrom) [inline app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt:103]: Use a pattern that is also found in other boolean sensors:

```suggestion
            if (connected) androidAutoConnected.statelessIcon else "mdi:car-off"
```

**Comment by jpelgrom** [comment]
You forgot to remove the old code after applying the suggestion...

**Comment by mikosoft83** [comment]
I'm new to the pull request game, how can I edit the commit?

**Comment by jpelgrom** [comment]
It's easier to make another commit with the changes if you're not familiar with Git :). (All commits will be merged into one by GitHub if your PR is accepted, so don't worry about it).

**Comment by mikosoft83** [comment]
I think I made it

**Review by jpelgrom — APPROVED** [review]

**Pull request merged**